Dataphor/DAE/Runtime/RuntimeException.cs

```diff
@@ -12,7 +12,7 @@ namespace Alphora.Dataphor.DAE.Runtime
 	using Alphora.Dataphor.DAE.Debug;
 	using Alphora.Dataphor.DAE.Runtime.Instructions;
 
-	public class RuntimeException : DAEException, ILocatorException
+	public class RuntimeException : DAEException, ILocatorException, ICodeContextException
 	{

```

Comment:
I don't see anywhere that would ever set the context of a RuntimeException? Am I missing it?